module_evaluations/TCR-6_2022-09-28.MD

```diff
@@ -0,0 +1,61 @@
+
+* [x] ACCEPTABLE
+* [x] ~INAPPLICABLE~
+* [ ] UNACCEPTABLE
+  * comments on what was evaluated/not evaluated, why a criterion failed
+
+## [Criteria](https://github.com/folio-org/tech-council/blob/4dc5ac6d632fb880131cd214d212f973683b9ebe/MODULE_ACCEPTANCE_CRITERIA.MD)
+
+* [X] Upon acceptance, code author(s) agree to have source code canonically in folio-org github
+* [X] Copyright assigned to OLF
+* [X] Uses Apache 2.0 license
+* [X] Third party dependencies use an Apache 2.0 compatible license
+* [X] Module’s repository includes a compliant Module Descriptor
+* [X] Modules must declare all consumed interfaces in the Module Descriptor “requires” and “optional” sections
+* [ ] Environment vars are documented in the ModuleDescriptor - _note: read more at [https://wiki.folio.org/pages/viewpage.action?pageId=65110683](https://wiki.folio.org/pages/viewpage.action?pageId=65110683)_
+  * The following documented environment variables are not present in the module descriptor: `EVENTS_PUBLISHER_BOOTSTRAP_SERVERS`, `EVENTS_CONSUMER_BOOTSTRAP_SERVERS`, `EVENTS_PUBLISHER_ZK_CONNECT`, `EVENTS_CONSUMER_ZK_CONNECT`
+* [X] Back-end modules must define endpoints consumable by other modules in the Module Descriptor “provides” section
+* [ ] All API endpoints are documented in RAML or OpenAPI
+  * This work is being done by the developers, but it is not yet complete.
+* [X] All API endpoints protected with appropriate permissions
+* [X] No excessive permissions granted to the module
+* [X] Code of Conduct statement in repository
+* [ ] Installation documentation included
+  * Documentation which assits an implementor in getting this module runnin locally would satisfy this requirement
+* [X] Contribution guide is included in repo
+* [X] Module provides reference data (if applicable)
+* [X] Personal data form is completed, accurate, and provided as PERSONAL_DATA_DISCLOSURE.md file
+* [X] Sensitive information is not checked into git repository
+* [X] Module is written in a language and framework that FOLIO development teams are familiar with _e.g. Vertx/RMB, Spring Way/folio-spring-base, and React/Stripes_
+  * Groovy is already in use in other FOLIO modules
+* [X] Back-end modules are based on Maven/JDK 11 and provide a Dockerfile
+* [ ] Integration (API) tests written in Karate if applicable -_note: these tests are defined in https://github.com/folio-org/folio-integration-tests_
+  * It is important that the itegration test are written in a technology that can be integrated esily into FOLIO's CI pipeline.
+* [ ] Back-end unit tests at 80% coverage
+  * There is sufficient branching logic within this implementation to strongly suggest that unit testing would be advantageous
+  * Integration tests might be used as a proxy for Unit tests, but only if they can produce a coverage report showing that > 80% of the code has been executed during the testing
```

Comment:
I thought it was agreed in the last TC meeting that if a calculation of coverage without generated code exceeded 80% this criteria would be passed.

Was that calculation made and the coverage was still less than 80%?

Reply:
This was the determination in the TC, at this time I have not received that calculation.

Reply:
Having not received the calculation, we decided that the module fails the criteria on this review.

Is that a sensible interpretation of the reasoning?